group/templates/group/groupDetail.html

```diff
@@ -0,0 +1,488 @@
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2014 NTHUOJ team
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<!DOCTYPE html>
+
+<head>
+  <meta charset="utf-8">
+  <title>National Ching-Hua University Online Judge</title>
+  <!-- Bootstrap&Somethings styles -->
+  <link rel="stylesheet" href="http://bootswatch.com/simplex/bootstrap.css" media="screen">
+  <link href="http://bootswatch.com/paper/bootstrap.css" rel="stylesheet">
+  <link rel="stylesheet" href="http://bootswatch.com/assets/css/bootswatch.min.css">
+
+  <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
+  <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
+  <!-- Magnific Popup core JS & CSS -->
+  <link rel="stylesheet" href="/css/magnific-popup.css">
+  <link rel="stylesheet" href="/css/mfp-fade.css">
+  <script src="/js/jquery.magnific-popup.js"></script>
+  <script src="/js/popup-setting.js"></script>
+</head>
+
+{% extends "group/base.html" %} {% load static %} {% block body_block %}
+
+<script language="javascript" type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/flot/0.8.2/jquery.flot.min.js"></script>
+<script language="javascript" type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/flot/0.8.2/jquery.flot.pie.min.js"></script>
+
+<div class="container-fluid">
+  <div class="row-fluid">
+    <a href="/list" class="btn btn-sm btn-default" data-toggle="modal" style="float:left">return</a>
+    <br>
+    <br> {% block Group_intro_block %}
+    <div class="span12">
+      <div class="hero-unit">
+        <div class="hero-unit">
+          <span style="width:50%; float:left">
+                            <p>Group name</p>
+                            <h2>{{ group_name }}</h2>
+                    </span>
+          <span style="width:50%; float:right">
+                            <p>Owner/TA</p>
```

Comment:
Html preetify done this, and I cannot understand either.
So I should change it to this form?
<div class="hero-unit">
+          <span style="width:50%; float:left">
+                <p>Group name</p>
+                <h2>{{ group_name }}</h2>
+         </span>